src/internal/EnhancedButton.js

```diff
@@ -117,9 +101,8 @@ const EnhancedButton = React.createClass({
     listenForTabPresses();
   },
 
-  componentWillReceiveProps(nextProps, nextContext) {
+  componentWillReceiveProps(nextProps) {
     this.setState({
-      muiTheme: nextContext.muiTheme || this.state.muiTheme,
```

Comment:
Um, uh, that isn't in mine. And mine was up-to-date with origin before I started fixing your suggestions! :confused: